app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,162 @@ import { useState } from "react";
 import Alert from "react-bootstrap/Alert";
 import Card from "react-bootstrap/Card";
 import Image from "react-bootstrap/Image";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import { BsZoomIn, BsZoomOut } from "react-icons/bs";
+import { SlActionUndo } from "react-icons/sl";
+import {
+  TransformWrapper,
+  TransformComponent,
+  useControls,
+} from "react-zoom-pan-pinch";
 import { Crash } from "@/types/crashes";
 
-// import axios from "axios";
-// import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
-
 const CR3_DIAGRAM_BASE_URL = process.env.NEXT_PUBLIC_CR3_DIAGRAM_BASE_URL!;
 
+const ZoomResetControls = ({
+  setRotation,
+}: {
+  setRotation: (value: number) => void;
+}) => {
+  const { zoomIn, zoomOut, resetTransform } = useControls();
+
+  const handleReset = () => {
+    resetTransform();
+    setRotation(0);
+  };
+
+  // pixel pushing
+  const iconStyle = {
+    position: "relative" as const,
+    top: "-1px",
+    fontSize: "1.3em",
+  };
```

Comment:
would you mind taking a look at our other icon + button components and matching their style? you should be able to get away with using zero `style` props here vs the utility classes—and the `AlignedLabel` component might be helpful too.

And ff you feel like we need a generic "IconButton" component—by all means go for it.

And if we're still unhappy with look + feel without pixel pushing, I am thinking we can add these kinds of todos in https://github.com/cityofaustin/atd-data-tech/issues/20013—so that we can hopefully solve them with SCSS in a centralized file 💅

Reply:
https://github.com/cityofaustin/vision-zero/pull/1635/commits/584d84f57f28e37f2be8a8aaea1dc4e632fd2be2

Thanks John -- I looked at buttons from around the app by searching for <Button> and I found plenty of examples where we had an pictograph + text, and because you mentioned `AligntedLabel`, I'm taking it to mean that you want some labels on these buttons. Right?

How's this look?